script/randomizer-ui-translator.js

```diff
@@ -70,6 +75,10 @@ const TRANSLATIONS = {
         "CHARACTER_EXECUTIONER": "Pyramid Head (Executioner)",
         "CHARACTER_BLIGHT": "Talbot Grimes (Blight)",
         "CHARACTER_TWINS": "Victor & Charlotte Deshayes",
+        "CHARACTER_ARTIST": "Carmina Mora (Artist)",
+        "CHARACTER_TRICKSTER": "Ji-Woon Hak (Trickster)":,
```

Comment:
You have a typo at the end of the line. This is probably the reason index.html was no longer working for you.

Should be: `"CHARACTER_TRICKSTER": "Ji-Woon Hak (Trickster)",`

I'll take a look on everything as soon as I can.

Cheers!
Norse